src/main/java/com/rakovets/course/java/core/practice/looping_statements/Task01.java

```diff
@@ -31,6 +31,10 @@ public static void main(String[] args) {
     static String generateNumbersColumn(int numberRows) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        String s = "";
```

Comment:
"Violation: Not informative variable name
Solution: Rename variable"

Example: `result`, `table`